### ISSUE TITLE
[Bug] Please upgrade compatibility with newer versions of dbt-utils

### ISSUE THREAD

**Issue by sgoley** [fivetran/dbt_netsuite#54]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

`fivetran/netsuite` depends on old versions of `dbt-utils` package.

### Relevant error log or model output

```shell
Version error for package dbt-labs/dbt_utils: Could not find satisfactory version from options
```

### Expected behavior

`fivetran/netsuite` does not continue to enforce requirement that dbt-utils version must be <0.9.0

### dbt Project configurations

# var configs
vars:
    netsuite_database: "SRC"
    netsuite_schema: "FIVETRAN_NETSUITE"

### Package versions

packages:
  - package: dbt-labs/codegen
    version: 0.8.1
  - package: emilyriederer/dbtplyr
    version: 0.3.2
  - package: fivetran/netsuite
    version: 0.6.2
  - package: calogica/dbt_expectations
    version: 0.5.8
  - package: dbt-labs/metrics
    version: 1.3.0

### What database are you using dbt with?

snowflake

### dbt Version

1.3.0

### Are you willing to open a PR to help address this issue?

- [ ] Yes.
- [X] Yes, but I will need assistance and will schedule time during our [office hours](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) for guidance
- [ ] No.

**Comment by fivetran-joemarkiewicz** [comment]
Hi @sgoley thanks for opening this issue.

This is actually something we are working to support across all of our packages. Unfortunately there are quite a large number of breaking changes that will impact our packages within the latest dbt-utils releases. Therefore, we have opted to skip the dbt-utils v0.9.0 release and plan to support the dbt-utils v1.0.0 version. We currently have PR #53 open which migrates the package to support the dbt-utils v1.0.0 version.

However, dbt-utils v1.0.0 is not out yet. dbt-labs has communicated that this should be live at the end of November. Once dbt-utils v1.0.0 is rolled out, we will be able to merge the PR mentioned and have support for the latest version.

**Comment by ropalloni** [comment]
It is out, when do you plan to merge?
https://hub.getdbt.com/dbt-labs/dbt_utils/latest/

**Comment by fivetran-joemarkiewicz** [comment]
Hi @ropalloni we merged the above mentioned PR #53 which offers support for the latest version of dbt-utils (v1.0.0) before our holiday break. We are now planning to cut an official release to then be live on the hub this week!

You can follow any other larger scale news around this upgrade for all our packages on the [tools-fivetran](https://getdbt.slack.com/archives/C01D1R2JLLA/p1671052979225199) channel within dbt Slack.

You should see this bug be addressed and officially released this week. 😄

**Comment by fivetran-joemarkiewicz** [comment]
Hi @sgoley the latest release of this package is now compatible with dbt-utils v1.0.0! As such, I will close this out 😄
